app/controllers/activities_controller.rb

```diff
@@ -0,0 +1,24 @@
+class ActivitiesController < ApplicationController
+
+  def empty 
```

Comment:
as we discussed that was my bad in task definition. We don't need that action in fact